### ISSUE TITLE
WOX not accessible with screen reader 

### ISSUE THREAD

**Issue by ghost** [Wox-launcher/Wox#530]
作者你好,我是一位視障人士需要用到SCREEN READER NVDA來操控電腦,我發現WOX當輸入了字串後,下面會列出各個項目,但無法以NVDA按down arrow 續一選擇其中一個項目,
唯一能做的就只有輸入完文字後直接enter

**Comment by bao-qian** [comment]
我用 windows 自带的 narrator 试了一下。
你说的是按了 down arrow 后，选中的东西没有读出来吧？
我是没写过 accessibility 的代码啦。。。
如果你能给点链接的话，会加快进度。。。

**Comment by ghost** [comment]
哦,是的,很感謝你的回覆
我用的是名叫nvda的發聲軟件,
http://www.nvaccess.org/

happlebao 於 24/3/2016 4:30 寫道:
